### DISCUSSION TITLE
Updating for Ashland and Bog Witch Updates

### DISCUSSION THREAD

**Discussion by Vapok** [Vapok/AdventureBackpacks#135] [Announcements]
# 1.7.9 - Additional Updates and Bug Fixes
* Updated Changes based on feedback from Pull Requests
* Fixed Bug that removed default effects from Backpacks
* Changed and updated default settings for Frost and Cold Resistances on Backpacks
  * You may need to reset your Cold Resist configs for Cold Resist to show back up on Lox Hide Backpack

# 1.7.8 - Forgot to increment the in-game version to 1.7.7
* In-game version now shows 1.7.8

# 1.7.7 - Updating for Ashland and Bog Witch Updates
* Updates to Valheim 0.219.14
* Updated ItemManager and Piece Manager

# 1.7.6 - Updating for 0.217.38 Valheim
* Fixed error on load.

# 1.7.5 - More Compatibility Support
* Added in Protection Override so that backpacks can be placed on an Item Stand, similar to Armor Stands.
* Added initial ContentsWithin support to prevent backpacks from being previewed. 

# 1.7.4 - One more Drop Bug
* Fixed: Drops were being enabled on world start.

# 1.7.3 - Fixed Configuration Issue
* Fixed the ItemManager issue and now running on new version.
* Drops tested and working as expected.

# 1.7.2 - Fixing Drop Issues
* Reverting to a previous version of ItemManager until I can understand why drops are not working fully.
  * If you having continued issues with drops after updating to this update, you might have reset your Adventure Backpacks configuration file.

<hr /><em>This discussion was created from the release <a href='https://github.com/Vapok/AdventureBackpacks/releases/tag/v1.7.9'>Updating for Ashland and Bog Witch Updates</a>.</em>

**Comment by mrcclassical** [comment]
Will there be a backpack released for the Ashlands?
